### PR TITLE
The phrase "need to want to" is grammatically incorrect

### PR DESCRIPTION
**Pull request by A-transformer**
The docstring contains "Set to true if you need to want to compute output logits/loss." The phrase "need to want to" is grammatically incorrect and unclear.

### CODE REVIEW

**Comment by A-transformer** [comment]
This is just a typo

**Comment by maanug-nv** [comment]
rerouted this change to `model_provider.py` and included in commit e002b5c23c0f8681bbf42df1c3eb07ec27ff5446
